### DISCUSSION TITLE
Take screenshot after every 5 min in all OS

### DISCUSSION THREAD

**Discussion by Aswathips** [AvaloniaUI/Avalonia#8068] [Q&A]
Hi,
I'm developing a cross-platform desktop application in which I want to take the screenshot of my desktop after every 5 min. By googling I get the code to take screenshot in windows OS.

                                       Rectangle bounds = Screen.GetBounds(Point.Empty);
                                       using(Bitmap bitmap = new Bitmap(bounds.Width, bounds.Height))
                                       {
                                            using(Graphics g = Graphics.FromImage(bitmap))
                                            {
                                                 g.CopyFromScreen(Point.Empty, Point.Empty, bounds.Size);
                                            }
                                            bitmap.Save("test.jpg", ImageFormat.Jpeg);
                                       }

But how can I make it possible in  Linux and Mac OS.
Anyone please help me.

**Comment by tofutim** [comment]
Did you ever figure it out? Seems like a hard problem! Probably need CoreGraphics in OSX and X11 interface in Linux.
